graphistry/feature_utils.py

```diff
@@ -292,7 +292,14 @@ def remove_internal_namespace_if_present(df: pd.DataFrame):
         config.IMPLICIT_NODE_ID,
         "index",  # in umap, we add as reindex
     ]
-    df = df.drop(columns=reserved_namespace, errors="ignore")  # type: ignore
+
+    if (len(df.columns) <= 2):
```

Comment:
great, can we get a `logger.debug` for this happening as it's not expected?

or better,

```python
class AutohealWarning(UserWarning)
  pass

...

  if len(df.columns) <= 2:
    warning.warn("...", AutohealWarning)
```